explorer/interpreter/value.cpp

```diff
@@ -524,6 +524,95 @@ void Value::Print(llvm::raw_ostream& out) const {
   }
 }
 
+auto ValueKindName(Value::Kind kind) -> std::string_view {
```

Comment:
If you want something like this, I'd suggest looking at what's being done on the toolchain side for enum stringification, or just printing the integer value. I don't think that we should maintain this kind of list.

https://github.com/carbon-language/carbon-lang/blob/trunk/toolchain/parser/parse_node_kind.h

Reply:
> If you want something like this, I'd suggest looking at what's being done on the toolchain side for enum stringification, or just printing the integer value. I don't think that we should maintain this kind of list.
> 
> https://github.com/carbon-language/carbon-lang/blob/trunk/toolchain/parser/parse_node_kind.h

Cool! Added value_kind.def and change enum definition and `ValueKindName` to use the new mechanism.